### PR TITLE
Adds compilation and debugging instructions

### LINKED ISSUE #4: Add compilation instructions
It sure ain't the easiest!

### LINKED ISSUE #5: Add debug instructions
Since it has to be done remotely.

### PR DESCRIPTION
**Pull request by Elekrisk**
Fixes #4, fixes #5

### CODE REVIEW

**Review by Vikdos — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -87,4 +87,34 @@ Any stage above `Usable` may or may not be feasible, depending on the scope of t
...
+
+### Compiling
+
+To build the project, run `cargo bootimage`
+
+### Running Debug kernel
+
+To run the project, `qemu-system-x86_64` must be installed.
+
+To run the project, run the command `qemu-system-x86_64 .\target\x86_64-unknown-caesarsallad\de\bootimage-brynolv-evper-ofjall-project.bin`.
```
Comment (brynolv) [inline README.md]: Beautiful line.

File: README.md [hunk 2]
```diff
@@ -87,4 +87,34 @@ Any stage above `Usable` may or may not be feasible, depending on the scope of t
...
+
+### Debugging Debug kernel
+
+To debug the project, `qemu-system-x86_64` and `gdb` must be installed.
+
+To debug the project, run the command `qemu-system-x86_64 .\target\x86_64-unknown-caesarsallad\de\bootimage-brynolv-evper-ofjall-project.bin -s`.
```
Comment (brynolv) [inline README.md]: Here as well.

File: README.md [hunk 3]
```diff
@@ -87,4 +87,34 @@ Any stage above `Usable` may or may not be feasible, depending on the scope of t
...
+
+### Running Debug kernel
+
+To run the project, `qemu-system-x86_64` must be installed.
+
+To run the project, run the command `qemu-system-x86_64 .\target\x86_64-unknown-caesarsallad\de\bootimage-brynolv-evper-ofjall-project.bin`.
+
+### Debugging Debug kernel
+
+To debug the project, `qemu-system-x86_64` and `gdb` must be installed.
```
Comment (brynolv) [inline README.md:115]: And where does one get `qemu`?

**Review by brynolv — APPROVED** [review]
lgtm

**Pull request merged**